cron/README.md

```diff
@@ -0,0 +1,11 @@
+# README
```

Comment:
The cron.go code will mostly get deleted in lieu of the PubSub work ongoing for Scorecard scaling, so lets hold off on this README for now. I'm yet to update GitHub issues with all details about PubSub solution, will try to get it done by EOD today.

On a different note, can we move k8s/cron.yaml inside cron/k8s folder (for that matter even the scripts/* which are connected to cron.go)? It would be nice to have all cron related code inside cron/ for better code readability. Wdyt?

Reply:
> The cron.go code will mostly get deleted in lieu of the PubSub work ongoing for Scorecard scaling, so lets hold off on this README for now. I'm yet to update GitHub issues with all details about PubSub solution, will try to get it done by EOD today.

Sounds good.


> On a different note, can we move k8s/cron.yaml inside cron/k8s folder (for that matter even the scripts/* which are connected to cron.go)? It would be nice to have all cron related code inside cron/ for better code readability. Wdyt?

I agree. Even I was thinking the same.

Reply:
> The cron.go code will mostly get deleted in lieu of the PubSub work ongoing for Scorecard scaling, so lets hold off on this README for now. I'm yet to update GitHub issues with all details about PubSub solution, will try to get it done by EOD today.

I am hoping it is going to be the proposed solution and not the actual code. 

If you are working on the code I would request you open a WIP PR.

Reply:
I will update with the proposed solution not code.

Reply:
Thanks!